### PR TITLE
feat: inlineBanner cosmos

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Adds Cosmos fixture for InlineBanner
- Includes DarkMode toggle button (WIP for styling the toggle button :) )

Screenshot 2023-05-19 at 18 32 50

### CODE REVIEW

File: src/common/pure/InlineBanner/cosmos.decorator.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { useCallback, useContext, ReactNode } from 'react'
+import { ThemeContext } from 'styled-components/macro'
+import { useDarkModeManager } from 'state/user/hooks'
+
+const DarkModeToggle = ({ children }: { children?: ReactNode }) => {
+  const theme = useContext(ThemeContext)
+  const [darkMode, toggleDarkModeAux] = useDarkModeManager()
+  const toggleDarkMode = useCallback(() => {
```
Comment (anxolin) [inline src/common/pure/InlineBanner/cosmos.decorator.tsx:8]: for this probably will be nicer to do something more global, like a decorator.

happy to help with this one, let me know!
https://github.com/react-cosmos/react-cosmos/blob/main/docs/README.md#decorators

**Review by anxolin — COMMENTED** [review]
Nice @fairlighteth !

File: src/common/pure/InlineBanner/cosmos.decorator.tsx (hunk 1 above)
Comment (fairlighteth) [reply src/common/pure/InlineBanner/cosmos.decorator.tsx:8]: A separate decorator than this one?

**Comment by elena-zh** [comment]
Hey @fairlighteth , great!
However, I don't see that all these banners support responsive mode: 
responsive mode

**Review by shoom3301 — APPROVED** [review]

**Pull request merged**